### DISCUSSION TITLE
[ERROR] Exception during resolving address: [Errno 2] Host name lookup failure

### DISCUSSION THREAD

**Discussion by rexund** [Security-Onion-Solutions/securityonion#10537] [General]
OS: Ubuntu 20.04
Install from repo v2.3.250
Master Node

I keep getting the error message: "[ERROR] Exception during resolving address: [Errno 2] Host name lookup failure" when I run Sudo soup or sudo bash so-network-setup. Both the operations complete, but I can't log into the SO console in my browser (I get the error, "Request failed with status code 500.")

I would really appreciate some help. There are only 3 entries in this knowledgebase that list the same error code, and they weren't helpful in resolving this. I've built scores of SO VMs, but I've migrated to QEMU-KVM for this build. I have this last hurdle to clear, but I've spent an embarrassing amount of time and resources on this build, and I'm going to have to give up if I can't solve this.

**Comment by cm-ops** [comment]
Are you able to resolve the hostname? What is your `url_base` in your global.sls? What are your salt grains for `fgdn`, `fqdn_ipv4`, and` fqdns`?

**Reply by rexund** [reply]
Regarding resolving the hostname, no:
$ ping securityonion
ping: securityonion: Name or service not known

Your other questions look interesting, but I don't know how to get at the answers. Are there some terminal commands I can run for: 
url_base in your global.sls
salt grains for fgdn, fqdn_ipv4, and fqdns

**Reply by rexund** [reply]
/etc/hosts
127.0.0.1   securityonion securityonion.localdomain localhost localhost.localdomain localhost6 localhost6.localdomain6
192.168.122.92  securityonion    ***[I added this to see if it would help f=resolve the hostname]
::1  securityonion securityonion.localdomain localhost localhost.localdomain localhost6 localhost6.localdomain6

/etc/hostname
securityonion

/etc/resolv.conf
nameserver 127.0.0.53
options edns0 trust-adetc
